mercury/mercury-library/protocol-issue-credential/src/main/scala/io/iohk/atala/mercury/protocol/issuecredential/IssueFormats.scala

```diff
@@ -0,0 +1,153 @@
+package io.iohk.atala.mercury.protocol.issuecredential
```

Comment:
I think it's better to create engineering documentation for this protocol instead of keeping it with the code.
What do you think, @FabioPinheiro?

Reply:
All the documentation is on https://github.com/decentralized-identity/waci-didcomm/blob/main/issue_credential/README.md
But I will try to improve on the other PR